packages/suite/src/components/suite/modals/AddAccount/components/AccountSelect.tsx

```diff
@@ -78,9 +83,11 @@ const AccountButton = (props: {
         enabled = false;
         description = 'Account index is greater than 10';
     }
-    // const accountType = getTypeForNetwork(props.network.accountType, props.intl);
-    const prefix = props.network.accountType ? `${props.network.accountType} ` : '';
-    const label = `${prefix}Account #${index}`;
+
+    const accountType = props.network.accountType
+        ? getTypeForNetwork(props.network.accountType)
```

Comment:
const accountType = getTypeForNetwork(props.network.accountType || 'normal');
instead of condition